to_do_list_part_two.py

```diff
@@ -23,18 +23,33 @@ def view_tasks(task_list):
         print("The to-do list is empty. Consider adding a new task!")
     else:
         print("To-Do List:")
-        for idx, task in enumerate(task_list, start=1):
-            print(f"{idx}. {task}")
+        priority_order = {'high': 1, 'medium': 2, 'low': 3}
+        sorted_tasks = sorted(task_list, key=lambda x: (priority_order[x['priority']], x['deadline']))
```

Comment:
In the document, this function is required to display all tasks **sorted by priority and deadline**.

```
3. View Tasks: Displays all tasks, sorted by priority and deadline.
```

This code only sort by priority and should be sorted by both priority and deadline.

example:
```
task_1, high, "2024-10-16"
task_2, high, "2024-10-20"
task_3, medium, "2024-10-16"
task_4, medium, "2024-10-17"
task_4, low, "2024-10-15"
```